### DISCUSSION TITLE
ReadToEnd error

### DISCUSSION THREAD

**Discussion by Jonathan818385** [Melledy/Grasscutter#2440] [Support]
In the console that opens with the game im getting an error 
"ReadToEnd: 0000000000000000
Failed to find ReadToEnd - Need to update
Hooked ReadToEnd - Original at: 0000000000000000"

Along with this im having an error 502 when attempting to log in.

**Comment by NotThorny** [comment]
This tells us that you haven't followed the setup guide to get the correct version of the game. You are getting 502 due to that, not due to anything with Grasscutter itself.

Please be sure to read the supported version listed in the setup guide, it is required to have this version.
https://github.com/Grasscutters/Grasscutter#quick-start-automatic

**Reply by lotusarthur** [reply]
。。。
